README.md

```diff
@@ -139,14 +133,13 @@ If you can't find your distro here, you will have to [build from source](https:/
 Details on how to setup and configure the bar and each module have been moved to [the wiki](https://github.com/polybar/polybar/wiki/Configuration).
 
 #### Install the example configuration
```

Comment:
The readme does not reflect this yet because this is not part of a release yet and I don't want new users installing the latest version (which won't have a default config until 3.6 is released) thinking there is a default config.

The readme will be updated, together with the wiki on the next minor release.